packages/http/src/models/Balance.ts

```diff
@@ -17,12 +17,14 @@ BigNumber.config({
 
 export default class Balance {
   public integerBalance: number
+  public floatBalance: number
   public type: CurrencyType
   private bigBalance: BigNumber
 
   constructor(integerBalance: number | undefined, type: CurrencyType) {
     this.integerBalance = integerBalance || 0
     this.bigBalance = new BigNumber(this.integerBalance)
+    this.floatBalance = this.bigBalance.times(type.coefficient).toNumber()
     this.type = type
   }
 
```

Comment:
maybe we can use `floatBalance` in `toString` in place of `const number = ...`

Reply:
Good call let me update